prime-router/metadata/fhir_mapping/hl7/datatypes/ED/EDExtension.yml

```diff
@@ -0,0 +1,51 @@
+url:
+  type: SYSTEM_URL
+  value: ed-attachment
+
+extension:
+  generateList: true
+  expressionType: nested
+  expressions:
+    - expressionType: resource
+      valueOf: datatypes/HD/ExtensionHD
+      specs: ED.1
+      constants:
+        extensionUrl: ED.1
+        isStringUrl: true
+    - expressionType: nested
+      vars:
+        ed2: STRING_ALL, ED.2
+        ed3: STRING_ALL, ED.3
+      condition: $ed3 NOT_NULL && $ed2 NOT_NULL
+      expressionsMap:
+        url:
+          type: STRING
+          value: ED.2
+        valueCodeableConcept:
+          valueOf: datatypes/ID/CodeableConcept
+          specs: ED.2
+          expressionType: resource
+    - expressionType: nested
+      vars:
+        ed4: STRING, ED.4
+      condition: $ed4 NOT_NULL
+      expressionsMap:
+        url:
+          type: STRING
+          value: ED.4
+        valueString:
+          valueOf: ED.4
+          expressionType: HL7Spec
+          type: STRING
+    - expressionType: nested
+      vars:
+        ed5: STRING, ED.5
+      condition: $ed5 NOT_EQUALS Base64
```

Comment:
Looks like this was the case before too, but I don't think we want this condition here. ED.4 is the one that would be `Base64`, not ED.5, but we might want to have this extension populated unconditionally. Hopefully that's straightforward enough that we can slip it in with this PR even though it's a little unrelated.

Reply:
Yep, that's correct.  I'll update the condition to just check for null